homeassistant/components/plex/sensor.py

```diff
@@ -99,11 +104,25 @@ def device_state_attributes(self):
     def update(self):
```

Comment:
Which method is doing the I/O here?

Reply:
So it seems like you can remove the whole `update` method and move that into `async_refresh_sensor`. Then instead of `self.async_schedule_update_ha_state(True)` you can call `self.async_write_ha_state()`

Reply:
Done. Only noticed one sync function that needed an executor wrapper.